scripts/rf_raw_request.py

```diff
@@ -27,6 +28,20 @@
 argget.add_argument( "--verbose", "-v", action = "store_true", help = "Indicates if HTTP response codes and headers are displayed", default = False )
 args = argget.parse_args()
 
+def ifmatch_header(redfish_obj, path, headers=None):
+    if headers is None:
+        headers = {}
+    try:
+        actions_search = "^/redfish/v1/.*/Actions/.*$"
```

Comment:
We can skip the action checking here; PUT and PATCH are not supported on actions, so trying to filter this out is just extra work.